src/print.js

```diff
@@ -0,0 +1,77 @@
+// const taskInput = document.getElementById('task');
```

Comment:
- [ ] Kindly never leave an unnecessary comment in your code, because at this moment your project is smaller and it is not causing an issue, but as you move on and have so many unnecessary comments, it will be difficult for you and for any other person to understand your code.

Reply:
Thank you for the feedback